flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/table/OverWindowITCase.scala

```diff
@@ -40,6 +40,11 @@ import scala.collection.mutable
 @RunWith(classOf[Parameterized])
 class OverWindowITCase(mode: StateBackendMode) extends StreamingWithStateTestBase(mode) {
 
+  @Before
+  def setupEnv(): Unit = {
+    env.getCheckpointConfig.disableUnalignedCheckpoints()
```

Comment:
Add a comment and link to a jira ticket why?

Reply:
Created two tickets (watermarks/rescale) and linked to them with a small comment.